### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by acesnik**
This may help guide users to include more info on initial error reports.

### CODE REVIEW

**Comment by acesnik** [comment]
This will make an option to choose an issue template when you choose "New Issue" within GitHub, similar to here https://github.com/smith-chem-wisc/Spritz/issues/new/choose

**Comment by acesnik** [comment]
Give it a try in Spritz if you want. Yep, it gives a few options of structure for the issue report. There's still a blank one, so that there's an option for an open-ended question if it's that.

**Comment by trishorts** [comment]
I got it. I love it. Let's work on it together!

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+name: Feature request
+about: Suggest an idea for MetaMorpheus
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the feature you'd like**
+What would you like to see done in MetaMorpheus?
```
Comment (trishorts) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Alternate text: "We love to create new features for our users. Please send us your great ideas and explain why they would be useful. Let us know if youhave data (real or synthetic) that we could use to test if our solution is achieving the desired result."

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Issue report
```
Comment (trishorts) [inline .github/ISSUE_TEMPLATE/issue_report.md:2]: Issue/Bug/Crash/Unexpected Result Report

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+assignees: ''
+
+---
+
+**Describing the issue**
+What issue did you encounter?
+
+**Reproducing the error**
+Which protein database did you use? What type of mass spectrometry data files did you use? If possible, please provide examples of the datafiles that reproduce the error. 
+
```
Comment (trishorts) [inline .github/ISSUE_TEMPLATE/issue_report.md:15]: We can provide a folder that you can use to upload the data. Let us know if the data needs to remain private.

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+**Expected behavior**
+If the issue has to do with interacting with the GUI, what did you expected to happen instead?
+
+**Screenshots**
+If applicable, pleas add screenshots to help explain the problem.
+
+**Desktop information**
+ - OS: [e.g. Windows 10]
+ - MetaMorpheus Version: [e.g. 0.0.320]
+ - Available RAM: [e.g. 32 GB]
```
Comment (trishorts) [inline .github/ISSUE_TEMPLATE/issue_report.md:27]: This information is contained in the Autogenerated manuscript prose file:

image

**Review by trishorts — COMMENTED** [review]
have a look at my suggestions. I'm excited to have this integrated into our github repo.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (acesnik) [reply .github/ISSUE_TEMPLATE/feature_request.md]: added it!

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 2 above)
Comment (acesnik) [reply .github/ISSUE_TEMPLATE/issue_report.md]: changed to something similar!

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 3 above)
Comment (acesnik) [reply .github/ISSUE_TEMPLATE/issue_report.md:15]: Noted!

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 4 above)
Comment (acesnik) [reply .github/ISSUE_TEMPLATE/issue_report.md:28]: Noted that they can find it in the prose file!

**Comment by acesnik** [comment]
Fantastic! Thanks for all the comments! 

You can also make direct edits to the files on github here and push them to this branch if you'd like to change anything else. https://github.com/smith-chem-wisc/MetaMorpheus/tree/issuetemplate/.github/ISSUE_TEMPLATE

**Review by trishorts — APPROVED** [review]
I think this is a fantastic addition to our repo! Thanks Anthony.

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Describing the issue**
+What issue did you encounter?
+
+**Reproducing the error**
+Which protein database did you use? What type of mass spectrometry data files did you use? If possible, please provide examples of the datafiles that reproduce the error. We can provide a folder that you can use to upload the data. Please let us know if the data needs to remain private.
```
Comment (leahvschaffer) [inline .github/ISSUE_TEMPLATE/issue_report.md:14]: Ask for .xaml method file?

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+What settings did you use? Did you use the defaults?
+
+**Expected behavior**
+If the issue has to do with interacting with the GUI, what did you expected to happen instead?
+
+**Screenshots**
+If applicable, pleas add screenshots to help explain the problem.
```
Comment (leahvschaffer) [inline .github/ISSUE_TEMPLATE/issue_report.md:22]: *please

**Review by leahvschaffer — APPROVED** [review]

**Review by Yuling1996 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 5 above)
Comment (rmmiller22) [reply .github/ISSUE_TEMPLATE/issue_report.md:14]: .toml file

**Comment by rmmiller22** [comment]
@acesnik When you address Leah's two comments (fix the spelling error and ask for .toml file to reproduce error) - I will merge it

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 5 above)
Comment (acesnik) [reply .github/ISSUE_TEMPLATE/issue_report.md]: added!

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 6 above)
Comment (acesnik) [reply .github/ISSUE_TEMPLATE/issue_report.md]: corrected, thanks!

**Comment by acesnik** [comment]
@rmmiller22 Done! Thanks for clarifying that it was `.toml` files we were looking for. :)

**Review by rmmiller22 — APPROVED** [review]

**Pull request merged**